gluster-block.spec.in

```diff
@@ -77,10 +77,14 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_unitdir}/gluster-blockd.service
 %{_unitdir}/gluster-block-target.service
 %else
-%attr(755, root, root) %{_initddir}/gluster-blockd
+%attr(0644,-,-) %{_initddir}/gluster-blockd
 %endif
 %config(noreplace) %{_sysconfdir}/sysconfig/gluster-blockd
-%{_sysconfdir}/sysconfig/gluster-block-caps.info
+%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block
+%dir %attr(0755,-,-) %{_sharedstatedir}/gluster-block/scripts
+     %attr(0755,-,-) %{_sharedstatedir}/gluster-block/scripts/wait-for-bricks.sh
```

Comment:
Executable scripts that do not change should be packaged (for Fedora based distributions) under `/usr/libexec/gluster-block/...`.

Reply:
Cool, in that case let me move this under '/usr/libexec/gluster-block/'